assets/src/scripts/monitoring-location/components/hydrograph/index.js

```diff
@@ -97,7 +97,8 @@ export const attachToNode = function (store,
             .then(() => {
                 // Fetch any extended data needed to set initial state
                 const currentParamCode = parameterCode ? parameterCode : getCurrentParmCd(store.getState());
-                if (period === 'P30D' || period === 'P1Y') {
+
+                if (period && period.substr(0,1) === 'P' && period !== 'P7D') {
```

Comment:
In general, we don't do checking on the hash parameters. Maybe we should but we don't anywhere else.

Reply:
I think I understand what you mean here, but I might be confused. Do you mean checking that 'period' is not null or undefined? I didn't do that first and ended up with a set of tests failing.

Reply:
So the test is testing to make sure nothing other than P30D or P1Y is passed. Now it needs to test PxxD or P1Y. So this check needs to change and the tests need to change

Reply:
Well, I see that I was missing out on excluding years other than P1Y, but I think that is fixed now.